examples/jest-puppeteer/package.json

```diff
@@ -5,15 +5,17 @@
   "license": "Apache-2.0",
   "scripts": {
     "postinstall": "(cd ../dummy-app && yarn install)",
-    "test": "jest"
+    "test": "yarn run test:build",
+    "test:build": "(cd ../../ && yarn build) && yarn run test:no-build",
+    "test:no-build": "jest"
   },
   "devDependencies": {
-    "@pollyjs/adapter-puppeteer": "*",
-    "@pollyjs/core": "*",
-    "@pollyjs/persister-fs": "*",
-    "jest": "*",
-    "jest-puppeteer": "*",
-    "puppeteer": "*",
-    "setup-polly-jest": "*"
+    "@pollyjs/adapter-puppeteer": "file:../../packages/@pollyjs/adapter-puppeteer",
```

Comment:
Can the polly packages not just `*`? Would that prevent us from running build?

Reply:
Not part of the workspace, would mean it would install them vs using the ones in the same project.  Adding the examples to the workspace though probably isn't worth it.  Cost of install goes up for something people probably rarely run (evident by how long they've been broken).

Reply:
Yeah, but the initial point of these example is that they are contained and can be taken out of the polly repo and still work as expected to as closely imitate real life use-cases as possible.

Reply:
I think we add a version range that we know is compatible with the example vs asterisk. This way we can update them at our leisure while keeping them stable.

Reply:
Sounds good to me 😄